extras/qibosoq_paper_runcards/README.MD

```diff
@@ -0,0 +1,18 @@
+# Qibosoq paper runcards
+
+In this folder, we have stored the runcards used for the experiments presented in the Qibosoq paper.
+
+For the timing experiments, we used the following runcards:
+- `routines_benchmarks.yml`
+- `scaling_benchmarks.yml`
+
+These experiments were conducted on a platform analogous to the one described in `my_platform.yml`.
+On the other hand, please note that the calibration experiments can only be exactly reproduced with the same hardware; therefore, the platform runcard used for those experiments is not provided.
```

Comment:
I'm not sure this is the best strategy: even though it is unstable, and not working outside the TII, having even that runcard could in principle help to understand what has been done.

In practice, I agree with you that it will change little to nothing.
But if anyone happens to arrive until here, a useless runcard looks better than an omitted one (but maybe this is just a personal perspective).